### PR TITLE
Update for the ci.ps1 script to enable logging of errors and update t…

### PR DESCRIPTION
**Pull request by izy-izzy**
Update for the script so it enables logging errors from response.
Update to documentation of the scripts so it is easier for user to understand.

https://deepcrawl.atlassian.net/browse/AUTO-978

### CODE REVIEW

File: docs/ci-scripts.md [hunk 1]
```diff
@@ -9,22 +9,56 @@ The Automator shell scripts allow your CI platform to start a crawl on a develop
...
+$env:AUTOMATOR_USER_KEY_ID = <number>
+$env:AUTOMATOR_USER_KEY_SECRET = <string>
+$env:AUTOMATOR_TEST_SUITE_ID = <string>
+$env:AUTOMATOR_START_ONLY = <boolean> | 1 | 0
+./ci.ps1
+```
+
+Example 2: 
+
+* replace **AUTOMATOR_TEST_SUITE_ID** with your id
```
Comment (rthiiyer82) [inline docs/ci-scripts.md:35]: Isn't this optional as mentioned above in line 16

File: ci.ps1 [hunk 2]
```diff
@@ -6,17 +6,21 @@ param(
 $secret = $env:AUTOMATOR_USER_KEY_SECRET
 $key_id = $env:AUTOMATOR_USER_KEY_ID
 
-Write-Output $testSuiteId
-
 function Get-Auth-Token {
+    $query = @{"query" = "mutation { createSessionUsingUserKey(input: {userKeyId:""$key_id"" secret:""$secret""}) { token }}"} | ConvertTo-Json -Depth 9
```
Comment (alek-wb) [inline ci.ps1:10]: Will double double-quotes work here?

Comment (alek-wb) [reply ci.ps1:10]: It's different to how it was before

**Review by alek-wb — DISMISSED** [review]
Looks OK

**Review by rthiiyer82 — COMMENTED** [review]
Minoir comment

File: ci.ps1 (hunk 2 above)
Comment (izy-izzy) [reply ci.ps1:11]: It actually works. And it should be this way as the windows PowerShell evaluates "" as "

File: docs/ci-scripts.md (hunk 1 above)
Comment (izy-izzy) [reply docs/ci-scripts.md:35]: Not for example 2.

**Review by rthiiyer82 — DISMISSED** [review]

**Review by alek-wb — DISMISSED** [review]
OK

**Review by rthiiyer82 — APPROVED** [review]

**Review by alek-wb — APPROVED** [review]
OK

**Pull request merged**